src/Uno.UI/Controls/BindableUIView.iOS.cs

```diff
@@ -93,12 +93,12 @@ private void Initialize()
 		/// <remarks>
 		/// The trick for this method is to move the child from one position to the other
 		/// without calling RemoveView and AddView. In this context, the only way to do this is
-		/// to call BringSubviewToFront, which is the only available method on UIView that manipulates 
+		/// to call BringSubviewToFront, which is the only available method on UIView that manipulates
 		/// the index of a view, even if it does not allow for specifying an index.
 		/// </remarks>
 		internal void MoveViewTo(int oldIndex, int newIndex)
 		{
-			var newShadow = _shadowChildren.ToList();
+			var newShadow = _shadowChildren.Materialized;
```

Comment:
This is highly dangerous: you are altering the list below
``` 
newShadow.RemoveAt(oldIndex);
newShadow.Insert(newIndex, view);
```
So if someone uses the `_shadowChildren.Materialized` in the `BringSubviewToFront` it will access a modified list ...

Reply:
I already found that, I already fixed it.